### PR TITLE
Deliver fonts and JavaScript from the XSF servers instead of Google

### PR DESCRIPTION
**Pull request by karoshi42**
To enhance the privacy of the site visitors, don't tell google they were there. Especially with 'Privacy-focused' being one of the first things which are said about XMPP (and for me definitely one main reason to use it), I think this might be an additional step to support that claim.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,5 @@ output/
 bower_components
 node_modules
 content/extensions
+.venv
+package-lock.json
```
Comment (guusdk) [inline .gitignore]: I don't think package-lock.json is supposed to go in .gitignore.

Comment (jcbrand) [reply .gitignore]: No, it should go into version control.

**Comment by Ppjet6** [comment]
It seems to me that the banner picture has been left out:

https://raw.githubusercontent.com/adambrault/xmpp.org-static/gh-pages/img/london.jpg

I'm not sure what is blocking this PR otherwise. I'd be happy to accept it as is as it's already an improvement over the status quo.

**Comment by Echolon** [comment]
@karoshi42 have you tried on this again?

I think this is good in terms of privacy.

**Comment by mwild1** [comment]
With the switch to Hugo (#953), all resources are now served from xmpp.org :+1:

**Pull request closed without merging**